src/container.c

```diff
@@ -151,6 +151,44 @@ static int container_setup_volume(struct hyper_container *container)
 	return 0;
 }
 
+static int container_setup_modules(struct hyper_container *container)
+{
+	struct stat st;
+
+	if (stat("./lib/modules", &st) < 0) {
+		if (errno != ENOENT) {
+			return -1;
+		}
+		hyper_mkdir("./lib/modules");
+	} else if (!S_ISDIR(st.st_mode)) {
+		return -1;
+	}
+
+	if (stat("./lib/modules/4.4.7-hyper", &st) == 0) {
```

Comment:
Can we get kernel version from other places like uname(2)? Just so that we don't have to modify it when updating kernel version in the future.